packages/design-system-tokens/src/themes/core.ts

```diff
@@ -31,7 +31,7 @@ export const themeColors: ColorTokens = {
   'error-dark':                 color['rose-600'],
   'error-darker':               color['rose-700'],
   'error-darkest':              color['rose-800'],
-  'error-light':                color['rose-200'],
+  'error-light':                color['rose-300'],
```

Comment:
I think this kind of further standardization should happen in a separate pull request into `main`.

Also if we're updating these colors, should we update each of the individual status color sections to also talk about the color shift in the semantic versions? Like perhaps the `## Red` section becomes `## Error / Red` and below the table that shows red disappearing, we have a table that shows `error-light` changing from `rose-200` to `rose-300`

Reply:
Cool, PR here! https://github.com/CMSgov/design-system/pull/2394